packaging/ubuntu-14.04/snapd.postinst

```diff
@@ -20,4 +20,6 @@ case "$1" in
             systemctl -f enable snapd.autoimport.service snapd.socket snapd.service snap.mount.service snapd.system-shutdown.service
             systemctl start snapd.autoimport.service snapd.socket snapd.service snap.mount.service
         fi
+
+	echo "Please reboot, logout/login or source /etc/profile to have /snap/bin added to PATH."
```

Comment:
That's a good point. With a bit of digging, I took inspiration from http://unix.stackexchange.com/questions/32041/checking-if-path-contains-home-mydir-and-adding-it-if-not-all-in-a-script/32054#32054 to do the search in PATH.